### PR TITLE
update and show assistant...note: no FE for show assistant

### PR DESCRIPTION
**Pull request by Thobeats**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,7 +4,7 @@ APP_KEY=
 APP_DEBUG=true
 APP_URL=https://dev.customerpay.me
 
-API_URL=https://dev.api.customerpay.me
+API_URL=https://dev.api.customerpay.mephp
```
Comment (eni4sure) [inline .env.example:7]: This is an error !!!!
Correct the URL to .PHP not .mephp

**Comment by eni4sure** [comment]
Correct this please

**Comment by eni4sure** [comment]
Also test your code before making a PR please...

File: .env.example (hunk 1 above)
Comment (Thobeats) [reply .env.example:7]: twas a mistake....alright

**Pull request closed without merging**